tools/clang/unittests/HLSL/ValidationTest.cpp

```diff
@@ -153,8 +153,6 @@ class ValidationTest : public ::testing::Test {
   TEST_METHOD(FunctionAttributes)
   TEST_METHOD(GSMainMissingAttributeFail)
   TEST_METHOD(GSOtherMissingAttributeFail)
-  TEST_METHOD(GSMissingMaxVertexCountFail)
-  TEST_METHOD(HSMissingPCFFail)
```

Comment:
These weren't validation tests, so glad to see them moved, but nor are `GSMainMissingAttributeFail` or `GSOtherMissingAttributeFail`.  A validation test is supposed to be a test of `DxilValidation`, not of clang diagnostics, which were typically tested in `VerifierTest` or newer lit shell test with `-verify`, not `ValidationTest`.  Now these are currently diagnosed during CodeGen, so it's understandable that they can't (yet) be moved to verifier, but they should have been FileCheck tests in the first place.

If there's a task to move these in a future PR, that's fine.  I just wanted to point this out.